pkg/util/util.go

```diff
@@ -185,3 +185,27 @@ func CanBeNonRoot(vmi *v1.VirtualMachineInstance) error {
 func MarkAsNonroot(vmi *v1.VirtualMachineInstance) {
 	vmi.Status.RuntimeUser = 107
 }
+
+func SetDefaultVolumeDisk(obj *v1.VirtualMachineInstance) {
+
+	diskAndFilesystemNames := make(map[string]struct{})
+
+	for _, disk := range obj.Spec.Domain.Devices.Disks {
+		diskAndFilesystemNames[disk.Name] = struct{}{}
+	}
+
+	for _, fs := range obj.Spec.Domain.Devices.Filesystems {
+		diskAndFilesystemNames[fs.Name] = struct{}{}
+	}
+
+	for _, volume := range obj.Spec.Volumes {
+		if _, foundDisk := diskAndFilesystemNames[volume.Name]; !foundDisk {
+			obj.Spec.Domain.Devices.Disks = append(
+				obj.Spec.Domain.Devices.Disks,
+				v1.Disk{
+					Name: volume.Name,
+				},
+			)
+		}
+	}
+}
```

Comment:
Would this one also fit the same file ?

Reply:
I wouldn't think so as it's not using anything from ClusterConfig. If anything it should be in a storage specific package but I couldn't find anything obvious when I looked. Happy to look again and move in a follow up if it's important.

Reply:
Yeah, you're right. I just looked for the word `volume` in that file and got no matches. 

I was just trying to avoid adding code to anything whose name is utils ... 

Is there any place related to storage that would fit @brybacki ? ...

If not, the code looks good, and you already have the lgtm tag. 

Thanks for listening :)

Reply:
there is no such place yet, at least I can not find it